static/stylesheets/scss/_alert.scss

```diff
@@ -40,3 +40,10 @@
   background-color: #fcf8e3;
   color: #8a6d3b;
 }
+
+.alert-important {
```

Comment:
Because we already have a red warning on every page, and I perceive that there's a diminishing return on figuring out how to hide it on this index page, I'd like to try a different color.

What do you think about a yellow similar to #d5ab73 from the banner we use for "old or pre-release version" ?

Reply:
👍 let me update the commit